common/test/acceptance/tests/studio/test_studio_settings_certificates.py

```diff
@@ -19,7 +19,7 @@ class CertificatesTest(StudioCourseTest):
     Tests for settings/certificates Page.
     """
     def setUp(self):  # pylint: disable=arguments-differ
-        super(CertificatesTest, self).setUp(is_staff=True)
+        super(CertificatesTest, self).setUp(is_staff=True, test_xss=False)
```

Comment:
This change was necessary because the test wants to ensure that a particular element is NOT present if display_org/display_coursenumber is not set. With test_xss as True, the display org and coursenumber values will always be set.